tests/jobs/design_kw/test_design_kw.py

```diff
@@ -78,6 +78,69 @@ def test_extract_key_value_dup_key():
         design_kw.extract_key_value(data)
 
 
+@pytest.mark.parametrize(
+    "str_or_dict, expected",
+    [
+        ("foo", "foo"),
+        ("foo:bar", "bar"),
+        ("foo:bar:com", "bar:com"),
+        ("", ""),
+        ({"foo:bar": "value"}, {"bar": "value"}),
+        ({"foo:bar": "value", "foo:com": "value2"}, {"bar": "value", "com": "value2"}),
+        ({}, {}),
+        # Duplicates are dropped, a warning is logged.
```

Comment:
I don't think any warning is logged here...

Reply:
Added extra duplicates warning.